This ensures that the Python and C++ bindings are **never** generated
in install(). The swig code doesn't work properly in a fakeroot and
running it in one causes very hard to debug build failures.

diff -upr mupdf-1.26.12.orig/Makefile mupdf-1.26.12/Makefile
--- mupdf-1.26.12.orig/Makefile	2025-12-19 18:05:13.211273367 +0100
+++ mupdf-1.26.12/Makefile	2025-12-19 18:05:43.421175855 +0100
@@ -637,7 +637,7 @@ endif
 
 install-shared-c: install-shared-check install-libs install-headers
 
-install-shared-c++: install-shared-c c++
+install-shared-c++: install-shared-c
 	install -m 644 platform/c++/include/mupdf/*.h $(DESTDIR)$(incdir)/mupdf
 	install -m $(SO_INSTALL_MODE) $(OUT)/libmupdfcpp.$(SO)$(SO_VERSION) $(DESTDIR)$(libdir)/
 ifneq ($(OS),OpenBSD)
@@ -645,7 +645,7 @@ ifneq ($(OS),OpenBSD)
 	ln -sf libmupdfcpp.$(SO)$(SO_VERSION) $(DESTDIR)$(libdir)/libmupdfcpp.$(SO)$(SO_VERSION_MAJOR)
 endif
 
-install-shared-python: install-shared-c++ python
+install-shared-python: install-shared-c++
 	install -d $(DESTDIR)$(pydir)/mupdf
 	install -m $(SO_INSTALL_MODE) $(OUT)/_mupdf.$(SO) $(DESTDIR)$(pydir)/mupdf
 	install -m 644 $(OUT)/mupdf.py $(DESTDIR)$(pydir)/mupdf/__init__.py
